### DISCUSSION TITLE
Old Pixel 1 device no longer can log in via official app or FolderSync

### DISCUSSION THREAD

**Discussion by bobloadmire** [nextcloud/all-in-one#4372] [Questions] [answered]
not sure if the latest nextcloud update broke my pixel 1 or something else, but I'm getting "error no response" and other timeout erros when trying to connect from my pixel 1. I can connect via webbrowser ok. but the app tells me to "update android webview" clicking the button takes me to the app store but it says app not found. additionally I'm trying to connect via FolderSync using the nextcloud sync option and this has never had an issue until now, now i'm getting timeout errors as well, i haven't changed anything in the configuration. Anyone have any ideas? I have a pixel 8 on the same wifi, no issues with anything. Rebooted the pixel 1 and set the wifi to defaults, cleared data and cache of all apps.

**Comment by szaimen** [comment, marked as answer]
> but the app tells me to "update android webview" clicking the button takes me to the app store but it says app not found.

Hi, this sounds like a bug in the android app. Feel free to open an issue here: https://github.com/nextcloud/android/issues
